src/kernels/webgl/binaryop_gpu.ts

```diff
@@ -52,7 +52,7 @@ export const POW = `
 if(a < 0.0 && floor(b) < b){
   return NAN;
 }
-return (round(mod(b, 2.0)) == 0 || round(mod(b, 2.0)) == 2) ?
+return (round(mod(b, 2.0)) != 1) ?
```

Comment:
@annxingyuan This change is outside the scope of the patch as it affects non packed ops. If needed I could take it out.